src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -14,12 +14,66 @@
 
 package com.google.sps.servlets;
 
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+
+import com.google.api.services.youtube.model.CommentThreadListResponse;
+import com.google.api.services.youtube.YouTube;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.Arrays;
+import java.util.Collection;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 
 /**
  * Servlet that fetch from Youtube Server.
  */
-@WebServlet("/userLogIn")
+@WebServlet("/YouTubeComments")
 public class YoutubeServlet extends HttpServlet {
+  private static final String DEVELOPER_KEY = "OUR_API_KEY"; //TODO: obtain actual API Key
+  private static final String APPLICATION_NAME = "SAY";
+  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+    throws IOException {
+    try {
+      YouTube youtubeService = getService();
+      String url = request.getParameter("url");
+      // Define and execute the API request
```

Comment:
Instead of commenting this, you can explain what the method does, for example, what is snippet and replies here?
Also since they seem to be hard coded, maybe move them with all the constants?

Reply:
This does not seem to be entirely resolved? Please update the comment. If you think there is nothing to add to the comment, then remove this comment as "Define and execute the API request" doesn't give any extra information.